primefaces/src/main/resources/META-INF/resources/primefaces/core/core.ajax.js

```diff
@@ -1348,12 +1348,11 @@ if (!PrimeFaces.ajax) {
 
                     var widget = PF(widgetVar);
                     if (widget && widget.isDetached() === true) {
-                        widget.destroy();
-
                         try {
+                            widget.destroy();
                             delete PrimeFaces.widgets[widgetVar];
-                            delete widget;
```

Comment:
why removing the "delete widget"?

Reply:
I changed it from `delete widget` which is no longer valid in modern browsers to `widget = null` to null out the variable.

Reply:
but delete PrimeFaces.widgets[widgetVar]; is still valid?

Reply:
yes you can delete JSON keys with `delete this[key]` but you can't delete a JSON object with `delete obj` it used to be an old IE hack but now all browsers want you to do `obj = null`